utils/utils.go

```diff
@@ -137,3 +138,17 @@ func ReadImageFromStaticPodDefinition(podFile, containerName string) (string, er
 
 	return "", fmt.Errorf("no '%s' container found or no image specified in %s", containerName, podFile)
 }
+
+func FindFilesInFolderAndRun(folder string, action func(string) error) error {
```

Comment:
It was my name proposition  and it is bad as all of them :) Maybe idea for better naming?

Reply:
I'm no better at proposing names, believe me :)

just a few that come from the top of my head, in no preference order (open to suggestions)
```
    WalkOverFiles
    RunOnEachFile
    ExecuteForFiles
    PerformOnFiles
```

Reply:
HandleFilesWithCallback